src/containers/LeftPanel.js

```diff
@@ -36,9 +36,9 @@ class LeftPanel extends Component {
 
   renderList() {
     const { documents, selected } = this.props
-    let filteredDocuments = this.state.searchText.length < 2 ? documents :
-      filteredDocuments = documents.filter((doc) => {
-        return doc.title.includes(this.state.searchText)
+    let filteredDocuments = this.state.searchText.length === '' ? documents :
```

Comment:
You're comparing number to empty string here (although both '' and 0 are falsey, you should still care about types)

Reply:
Hm, it doesn't work when just this.state.searchText.length ?